### PR TITLE
add Design and navigate in AddWorkoutPage

### PR DESCRIPTION
**Pull request by betulEZ**

### CODE REVIEW

File: frontend/src/pages/AddWorkoutPage.tsx [hunk 1]
```diff
@@ -33,6 +35,7 @@ export default function AddWorkoutPage() {
             await axios.post('/api/workouts', formData);
             setFormData({ name: '', description: '' });
             alert(`Thanks. Workout "${formData.name}" added.`);
+            navigate("/");
```
Comment (Quarzerback) [inline frontend/src/pages/AddWorkoutPage.tsx:38]: no navigate here. after adding a workout, we should stay on the adding-page. only after edit, we want to go to the details page

File: frontend/src/pages/EditWorkoutPage.tsx [hunk 2]
```diff
@@ -32,14 +32,16 @@ export default function EditWorkoutPage(props: Props) {
             })
                 .then(props.fetchData);
 
-                    navigate("/workouts");
+                    navigate("/");
```
Comment (Quarzerback) [inline frontend/src/pages/EditWorkoutPage.tsx:35]: after editing, we want to go to the details-page of the workout, not to home

File: frontend/src/pages/HomePage.tsx [hunk 3]
```diff
@@ -2,6 +2,7 @@ import {Workout} from "../types/Workout.ts";
 import {useEffect} from "react";
 import axios from "axios";
 import {Link} from "react-router-dom";
+import "/src/App.css"
```
Comment (Quarzerback) [inline frontend/src/pages/HomePage.tsx:5]: i think we should do the styling in a css-file for each component. in app.css there should only be stylings that adress everything - and i dont think, we need to import it then.

File: frontend/src/pages/EditWorkoutPage.tsx [hunk 4]
```diff
@@ -32,14 +32,16 @@ export default function EditWorkoutPage(props: Props) {
...
         }
     }
 
     return (
         <>
+            <div style={{textAlign:"center"}}>
```
Comment (Quarzerback) [inline frontend/src/pages/EditWorkoutPage.tsx:41]: I'm not sure, but shouldn't we try to do all of the styling in the css-files and not directly in the tsx?

File: frontend/src/pages/DetailsPage.tsx [hunk 5]
```diff
@@ -1,6 +1,7 @@
 import {Workout} from "../types/Workout.ts";
 import {useNavigate, useParams} from "react-router-dom";
 import axios from "axios";
+import "/src/App.css"
```
Comment (Quarzerback) [inline frontend/src/pages/DetailsPage.tsx:4]: i think we should do the styling in a css-file for each component. in app.css there should only be stylings that adress everything - and i dont think, we need to import it then.

File: frontend/src/pages/HomePage.tsx (hunk 3 above)
Comment (betulEZ) [reply frontend/src/pages/HomePage.tsx]: You're right, that's not necessary.

**Review by Quarzerback — APPROVED** [review]

**Pull request merged**